engine/function/src/templates/Numeric.ftl

```diff
@@ -525,7 +525,7 @@ public class Numeric {
 
         final long n = values.size();
 
-        if(n != weights.size()){
+        if (n != weights.size()){
```

Comment:
I'm sorry, can you fix all of these also? I tried to mention it in my previous comment but I got the punctuation wrong.
Basically change `){` to `) {`